src/client/swagger/unity.yml

```diff
@@ -23,174 +23,197 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/region':
+  '/marketplace':
     get:
-      operationId: GetRegion
+      operationId: GetMarketplace
       tags:
-        - Region
+        - Marketplace
       responses:
         '200':
-          description: Account region for billing
+          description: Marketplace information for account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Region'
+                $ref: '#/components/schemas/Marketplace'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/account/history':
+  '/billing/start_date':
     get:
-      operationId: GetHistory
+      operationId: GetBillingStartDate
       tags:
-        - History
+        - Billing
       responses:
         '200':
-          description: Account history for billing, rates and usage
+          description: Start of the current billing period for the Account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/History'
+                $ref: '#/components/schemas/BillingDate'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/notify_settings':
+  '/usage/vectors':
     get:
-      operationId: GetBillingNotifySettings
+      operationId: GetUsageVectors
       tags:
-        - BillingNotifySettings
+        - Usage
       responses:
         '200':
-          description: Settings for billing notifications
+          description: Usage vectors for the account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BillingNotifySettings'
+                $ref: '#/components/schemas/UsageVectors'
+
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-    put:
-      operationId: PutBillingContact
+  '/usage/{type}':
+    get:
+      operationId: GetUsagetype
       tags:
-        - PaymentMethods
-      requestBody:
-        description: Billing Contact to Update
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/BillingNotifySettings'
+        - Usage
+      parameters:
+        - in: path
+          name: type
+          required: true
+          schema:
+            type: string
+            enum: [limits, billing, storage_gb, write_mb, reads_gb, query_count]
+          description: The usage type or vector to query. Can be a usage metric, billing stats, or rate limit events
```

Comment:
Are type and vector synonyms? If so, we should prob pick one or the other for these two endpoints.

Reply:
Will replace `type` with `vector_name`